backends/ebpf/target.cpp

```diff
@@ -20,6 +20,21 @@ limitations under the License.
 namespace EBPF {
 
 
+void Target::emitPreamble(Util::SourceCodeBuilder* builder) const {
+    (void) builder;
+}
+
+void Target::emitTraceMessage(Util::SourceCodeBuilder* builder, const char* format,
```

Comment:
you can perhaps document this function explaining what the arguments represent.
Also, you can probably give (in comments) an example where you use runtime arguments; there are such examples in the code, but perhaps here they would be easier to find.